StarWars5e.Parser/Parsers/BaseProcessor.cs

```diff
@@ -25,7 +25,7 @@ private static async Task<List<string>> ReadInternalFile(IEnumerable<string> loc
 
                 foreach (var location in locations)
                 {
-                    using (var stream = Assembly.GetEntryAssembly()
+                    using (var stream = Assembly.GetExecutingAssembly()
```

Comment:
Let me know if this will be a problem. When running unit tests, the entry assembly is some unit test dll, rather than this dll.

I figured this change is probably safe however.

Reply:
Yeah, this is fine.